Makefile.js

```diff
@@ -236,7 +236,7 @@ function commitSiteToGit(tag) {
         exec(`git tag ${tag}`);
     }
 
-    exec("git fetch origin && git rebase origin/master");
+    exec("git fetch origin && git rebase origin/main");
```

Comment:
```suggestion
    exec("git fetch origin && git rebase origin/master");
```

This part of the code works with https://github.com/eslint/website, where the branch is still `master`.

Reply:
Thanks for review @mdebruijne